app/src/components/map/Timebar.jsx

```diff
@@ -113,14 +127,22 @@ class Timeline extends Component {
       .attr('class', css['c-timeline-inner-brush'])
       .call(this.innerBrushFunc);
 
-    // mmove both brushes to initial position
-    this.outerBrushFunc.move(this.outerBrush, [0, width]);
-    this.redrawInnerBrush(this.props.filters.timelineInnerExtent);
-
     // no need to keep brush overlays (the invisible interactive zone outside of the brush)
     this.outerBrush.select('.overlay').remove();
     this.outerBrush.select('.selection').attr('cursor', 'default');
+    this.outerBrush.selectAll('.selection').classed(css['c-timeline-outer-brush-selection'], true);
     this.innerBrush.select('.overlay').remove();
+    this.innerBrush.select('.selection').classed(css['c-timeline-inner-brush-selection'], true);
+    innerBrushLeftCircle = this.innerBrush.append('circle');
+    innerBrushRightCircle = this.innerBrush.append('circle');
+    this.innerBrush.selectAll('circle')
+      .attr('cy', height / 2)
+      .attr('r', 5) // this would be better in the css but Safari is stupid
```

Comment:
It is valid since SVG 2 https://www.w3.org/TR/SVG2/styling.html#PresentationAttributes
But I'll remove that comment, there are no stupid browsers, only lazy developers :D
(and thanks for testing in Safari BTW)